library/named-operation-library/src/test/java/uk/gov/gchq/gaffer/named/operation/handler/NamedOperationHandlerTest.java

```diff
@@ -216,7 +216,7 @@ public void shouldExecuteNamedOperationWithParameter() throws OperationException
         final NamedOperationDetail extendedNamedOperation = new NamedOperationDetail.Builder()
                 .operationName(opName)
                 .description("standard operation")
-                .operationChain("{ \"operations\": [ { \"class\":\"uk.gov.gchq.gaffer.operation.impl.get.GetAllElements\" }, { \"class\":\"uk.gov.gchq.gaffer.operation.impl.Limit\", \"resultLimit\": \"${param1}\" } ] }")
+                .operationChain("{ \"class\":\"uk.gov.gchq.gaffer.operation.OperationChain\", \"operations\": [ { \"class\":\"uk.gov.gchq.gaffer.operation.impl.get.GetAllElements\" }, { \"class\":\"uk.gov.gchq.gaffer.operation.impl.Limit\", \"resultLimit\": \"${param1}\" } ] }")
```

Comment:
Oh, it looks like this is going to require a migration for any existing named operations. Could we use something like the OperationChainDAO here instead, so the class name isn't required?